FsAutoComplete/test/integration/Format/output.json

```diff
@@ -0,0 +1,61 @@
+{
+  "Kind": "project",
+  "Data": {
+    "Project": "<absolute path removed>/test/integration/Format/Format.fsproj",
+    "Files": [
+      "<absolute path removed>/test/integration/Format/okbydefault.fs",
+      "<absolute path removed>/test/integration/Format/defaultoffset.fs",
+      "<absolute path removed>/test/integration/Format/offset5.fs"
+    ],
+    "Output": "<absolute path removed>/test/integration/Format/bin/Debug/Test1.exe",
+    "References": [
+      "<absolute path removed>/FSharp.Core.dll",
+      "<absolute path removed>/System.Core.dll",
+      "<absolute path removed>/System.dll",
+      "<absolute path removed>/mscorlib.dll"
+    ],
+    "Logs": {
+      "<absolute path removed>/test/integration/Format/Format.fsproj": ""
+    }
+  }
+}
+{
+  "Kind": "info",
+  "Data": "Synchronous parsing started"
+}
+{
+  "Kind": "errors",
+  "Data": []
+}
+{
+  "Kind": "format",
+  "Data": "namespace Ok\r\n\r\nmodule Test1 = \r\n    let foo = \"ok\"\r\n"
+}
+{
+  "Kind": "info",
+  "Data": "Synchronous parsing started"
+}
+{
+  "Kind": "errors",
+  "Data": []
+}
+{
+  "Kind": "format",
+  "Data": "namespace Ok\r\n\r\nmodule Test3 = \r\n    let foo = \"ok\"\r\n"
+}
+{
+  "Kind": "info",
+  "Data": "Synchronous parsing started"
+}
+{
+  "Kind": "errors",
+  "Data": []
+}
+{
+  "Kind": "format",
+  "Data": "namespace Ok\r\n\r\nmodule Test2 = \r\n     let foo = \"ok\"\r\n"
+}
+{
+  "Kind": "format",
+  "Data": "namespace Ok\r\n\r\nmodule Test2 = \r\n     let foo = \"ok\""
```

Comment:
These `\r\n` mean that the tests will probably fail on Linux. The easiest is probably to have `'n` here and extend the normalization function slightly to do that mapping.